libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -1,29 +1,32 @@
 import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
-import { Store } from '@ngrx/store';
+import { ActionsSubject, Store } from '@ngrx/store';
 import {
   addToReadingList,
   clearSearch,
   getAllBooks,
+  getReadingList,
   ReadingListBook,
   searchBooks
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
-import { Book } from '@tmo/shared/models';
-import { Observable } from 'rxjs';
-
+import { Book, ReadingListItem } from '@tmo/shared/models';
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
   styleUrls: ['./book-search.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class BookSearchComponent implements OnInit {
-  books$: Observable<ReadingListBook[]>;
+  books: ReadingListBook[];
+  books$ = this.store.select(getAllBooks);
```

Comment:
Keep it like `books$: Observable<ReadingListBook[]> = this.store.select(getAllBooks);`

Reply:
Changes are done.